mmv1/third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -220,7 +220,7 @@ func isBigqueryIAMQuotaError(err error) (bool, string) {
 // exceeding the quota limit for 'OperationReadGroup'
 func isOperationReadQuotaError(err error) (bool, string) {
 	if gerr, ok := err.(*googleapi.Error); ok {
-		if gerr.Code == 403 && strings.Contains(gerr.Body, "Quota exceeded for quota group 'OperationReadGroup'") {
+		if gerr.Code == 403 && strings.Contains(gerr.Body, "Quota exceeded for quota group ") {
```

Comment:
```suggestion
		if gerr.Code == 403 && strings.Contains(gerr.Body, "Quota exceeded for quota group") {
```
let's go ahead and trim the last space, just in case